server/api/themoviedb/index.ts

```diff
@@ -351,6 +356,8 @@ class TheMovieDb extends ExternalAPI {
           page,
           include_adult: includeAdult,
           language,
+          with_release_type: 3,
```

Comment:
Release type 3 is 'Theatrical.' Would it make sense to add any other types as well? Specifically 1 Premiere and 2
Theatrical (limited)